README.md

```diff
@@ -26,11 +26,17 @@ Kjør appen i utviklingsmodus:
 yarn start
 ```
 
+Bygging av appen i utviklingsmodus:
+
+```bash
+yarn build
```

Comment:
Hmm, yarn build lager et prod bygg, mente du yarn start?

Reply:
nei, ble nokk feil her.

Eg tenkte egenlig å skrive, hvilke kommandoer bør man kjøre, for å verifiser at bygget er ok
før man evt pusher.

Hadde dette passet?

`Bygging av appen

```bash
yarn build
```
`

Reply:
Ja du kan jo f.eks skrive noe som:

```
Verifisering av appen lokalt:
* lint: `yarn lint`
* sjekk typescript: `yarn tsc`
* kjør tester: `yarn test`
* bygg appen: `yarn build`
```

Reply:
Enig